localstack/services/dynamodb/resource_providers/aws_dynamodb_table.py

```diff
@@ -231,6 +231,12 @@ def create(
                     **self.get_ddb_kinesis_stream_specification(model)
                 )
 
+            if model.get("TimeToLiveSpecification"):
+                request.aws_client_factory.dynamodb.update_time_to_live(
+                    TableName=model["TableName"],
+                    TimeToLiveSpecification=model["TimeToLiveSpecification"],
+                )
+
```

Comment:
Just to check: I assume this operation can be done on a table that is still being created? Otherwise we should delay this call until e.g. (new) line 257

Reply:
Good point. I moved the code down below. AWS triggers a `ResourceNotFoundException` if the table is not yet there.